File: sdk/javascript-closure-angular/API/Client/RealTimeInformationList.js
goog.provide('API.Client.RealTimeInformationList');

/**
 * @record
 */
API.Client.RealTimeInformationList = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.RealTimeInformationList.prototype.errorcode;

/**
 * @type {!string}
 * @export
 */
API.Client.RealTimeInformationList.prototype.errormessage;

/**
 * @type {!number}
 * @export
 */
API.Client.RealTimeInformationList.prototype.numberofresults;

/**
 * @type {!string}
 * @export
 */
API.Client.RealTimeInformationList.prototype.stopid;

/**
 * @type {!string}
 * @export
 */
API.Client.RealTimeInformationList.prototype.timestamp;

/**
 * @type {!Array<!API.Client.RealTimeInformation>}
 * @export
 */
API.Client.RealTimeInformationList.prototype.results;

File: sdk/async-scala/src/main/scala/io/swagger/client/model/RealTimeInformationList.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class RealTimeInformationList (
  errorcode: Option[String],
errormessage: Option[String],
numberofresults: Option[Integer],
stopid: Option[String],
timestamp: Option[String],
results: Option[List[RealTimeInformation]]
)
